### DISCUSSION TITLE
Changing the window and curser positioning in refresh_output_panel function

### DISCUSSION THREAD

**Discussion by rorymullen** [yaroslavyaroslav/OpenAI-sublime-text#24] [Ideas]
Thanks for this fantastic package. Now that I'm using the package more, I find myself often using the ```OpenAI: Refresh Chat``` action. 

When I use this action (on my M1 mac running Sublime Test build 4152), the chat window opens to the top of the chat, rather than the bottom. I describe the issue in more detail [here](https://github.com/yaroslavyaroslav/OpenAI-sublime-text/issues/18#issuecomment-1722422944).  

**Idea: Might it be useful to open the output window at the bottom, to show the most recent message?** Might it also be useful to move the cursor to the output window, so that GPT responses can easily be copied? I'm thinking about a mouse-free workflow, where the ```OpenAI: Refresh Chat``` is key bound, and the output window can be navigated by keyboard to copy sections of the GPT response. 

If this behavior sounds like it would be attractive to users, then I think it can be easily implemented by adding two lines to end of the ```refresh_output_panel``` function, 

```
    output_panel.show(output_panel.size())  # Scroll to the end
    window.focus_view(output_panel)  # Move cursor to output panel
```
I haven't tested this though. For reference, the ```refresh_output_panel``` function is defined in the outputpanel.py file in the project, as follows
```
    def refresh_output_panel(self, window):
        output_panel = self.__get_output_panel__(window=window)
        output_panel.set_read_only(False)
        self.clear_output_panel(window)

        for line in Cacher().read_all():
            if line['role'] == 'user':
                output_panel.run_command('append', {'characters': f'\n\n## Question\n\n'})
            elif line['role'] == 'assistant':
                ## This one placed here as there're could be loooong questions.
                output_panel.run_command('append', {'characters': '\n\n## Answer\n\n'})

            output_panel.run_command('append', {'characters': line['content']})

        output_panel.set_read_only(True)
```

This is just an idea, thank you again for putting this package together!

**Comment by yaroslavyaroslav** [comment]
Hi, thanks a lot for your support and for all these kind words you say!

This is definitely make sense to make such thing, more than that it was the behavior that was implemented in its previous version, that were less useful though and was made that way not by purpose but for effort reduction. 

But yep, I believe this could be turned back in an easy way, I believe this would be made and released within a week or two at its worst, depending on my main job workload. 

Just to expand this discussion a bit, I wonder to implement a some kind of table of contents for this panel after all, of course an interactive one, coz such predefined (un)scrolling option is nothing but a halfway solution after all, but yet I saw no sights that there's an easy way to do that with given Sublime API, so it's just a sweet talk for a foreseeable future.
